src/Acuminator/Acuminator.Analyzers/StaticAnalysis/EventHandlers/EventHandlerAnalyzer.cs

```diff
@@ -75,17 +75,17 @@ protected override void AnalyzeSymbol(SymbolAnalysisContext context, PXContext p
 			});
 		}
 
-		private void AnalyzeLambda(OperationAnalysisContext context, PXContext pxContext)
-		{
-			if (context.Operation is ILambdaExpression lambdaExpression)
-			{
-				var symbolAnalysisContext =
-					new SymbolAnalysisContext(lambdaExpression.Signature, context.Compilation, context.Options,
-											  context.ReportDiagnostic, d => true, // this check is covered inside context.ReportDiagnostic
-											  context.CancellationToken);
+		//private void AnalyzeLambda(OperationAnalysisContext context, PXContext pxContext)
```

Comment:
Why did you comment these lines?
I propose delete them if they are unnecessary.

Reply:
In new Roslyn this API is not found. I think we should later look where this API went and check if it is deprecated or just has a different name

Reply:
Ok, but I think we need to figure it out before we will merge this branch to be sure that we will merge stable branch

Reply:
This pull request goes into internal branch `feature/ATR-382-dev-VS-UPGRADE` which I created as a feature branch to work on upgrade. It will be merged after the 1.6.2 release. 
The AnalyzeLambda API wasn't used before in Acuminator - it was added by @dropsonic  and as far as I remember it required some sort of switch in Roslyn or didn't worked with Visual Studio. I remember that it was some kind of experimental feature in the version of Roslyn that we used.
It was stored to be turned on after the upgrade of the Roslyn

Reply:
Ok, in that case we need to figure it out before merging it to the `dev` branch

Reply:
This part was not "experimental", it covers some valid cases with lambdas. But indeed we need to look into changes in the new Roslyn API.